layouts/partials/engage/breadcrumbs.html

```diff
@@ -0,0 +1,35 @@
+{{- $baseUrl := .Site.BaseURL -}}
+<nav class="breadcrumb-navigation">
+	<ul class="breadcrumb">
+		<li itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
+			<a itemprop="url" href="{{$baseUrl}}">
+				<i class="fa fa-home" aria-hidden="true"></i>
+				<span itemprop="title">Home</span>
+			</a>
+		</li>
+		{{- if not (eq .Layout "home") -}}
```

Comment:
I think this check is overkill as you already checking inside:

```
if layout is NOT home

  if layout is channel
  else 

  if layout is channelVideo
```

You can do just:

```
if layout is channel
  else 

if layout is channelVideo
```